frontend/src/views/MapView.vue

```diff
@@ -26,6 +26,16 @@
       </v-col>
     </v-row>
   </v-container>
+    <v-btn
+      @click="toggleModelSelectDrawer"
+      width="100%"
+      class="mb-2"
+    >
+      <v-icon
+        class="ma-1"
+        :icon="modelsStore.selectedModel ? mdiCheckCircleOutline : mdiNumeric1CircleOutline"
+      ></v-icon>
+    </v-btn>
```

Comment:
I think this was added on accident? looks like you were experimenting and it got committed on accident
I removed it